### ISSUE TITLE
East Asia is not in the list of IPs to allow for inbound connections.

### ISSUE THREAD

**Issue by tomaeda** [MicrosoftDocs/azure-devops-docs#13138]
I believe that when an asian user creates an organization, the east asia or southeast asia region is chosen.
However, the list of IPs to allow for inbound connections is only Singapore in southeast asia, and Hong Kong in east asia is not on the list.

Previously, whenever user created an organization in Asia, southeast asia was always used.
Now east asia may be chosen.
For example, if the user's country is Japan, east asia is selected.
Could you please add east asia to the list of IPs that should be allowed for inbound connections.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: dc5a4cfa-aa84-9f0f-55f4-71ec4861c070
* Version Independent ID: d621620a-c924-03b1-8b43-f99c33e58bb8
* Content: [Allowed address lists and network connections - Azure DevOps](https://learn.microsoft.com/en-us/azure/devops/organizations/security/allow-list-ip-url?tabs=IP-V4&view=azure-devops)
* Content Source: [docs/organizations/security/allow-list-ip-url.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/organizations/security/allow-list-ip-url.md)
* Service: **azure-devops**
* Sub-service: **azure-devops-security**
* GitHub Login: @chcomley
* Microsoft Alias: **chcomley**

**Comment by chcomley** [comment]
Hi @tomaeda, thank you for reaching out.  We no longer support creating organizations in East Asia, so it's removed from the list of locations. Please let me know if you have any further questions. You can @mention me. Thanks.

**Comment by tomaeda** [comment]
Hello, @chcomley, thank you for your reply.
I looked at the organization creation form again today. I confirmed that I can't create an organization in east asia.
However, users previously could create organizations in east asia. And some users are still using it in east asia.

I understand that it is not a bug in the documentation.
In that case, I think the issue is that there are users in east asia, so I'll consider giving feedback to Azure DevOps.

Example: 
![image](https://user-images.githubusercontent.com/79292498/221456424-1235fbee-99be-4c8a-abc2-f11690e2998f.png)

This is one of the organizations I created. This organization is still in the east asia region.
